index.bs

```diff
@@ -138,8 +147,29 @@ intersection with the <a>intersection root</a>, as per the
 The IntersectionObserver interface</h3>
 
 The {{IntersectionObserver}} interface can be used to observe changes in the
-intersection of a <a>target</a> {{Element}} and a {{IntersectionObserver/root}}
-{{Element}} (or the top-level document's viewport).
+intersection of a {{IntersectionObserver/root}} {{Element}} (or the
+<a>top-level browsing context</a>'s visual viewport), and one or more <a>target</a> {{Elements}}.
```

Comment:
Elements point to https://dom.spec.whatwg.org/#elements which is an array-like collection.